### PR TITLE
Issue #175 support deno doc

### PR DESCRIPTION
**Pull request by crookse**
Addresses #175 

This is a breaking change PR. Drash will now need to be imported like ...

```
import { Drash } from "https://deno.land/x/drash/mod.ts";
```

instead of ...

```
import Drash from "https://deno.land/x/drash/mod.ts";
```

### CODE REVIEW

**Comment by ebebbington** [comment]
On the commit `[issue-#175-support-deno-doc] test all of the things` (`
3436713`), because the multiple commands to run the tests aren't running if the previous one succeeds (eg `&&`), if the first test command fails and the second one passed wont this mean (for CI) that the tests have passed?

**Comment by crookse** [comment]
> On the commit `[issue-#175-support-deno-doc] test all of the things` (` 3436713`), because the multiple commands to run the tests aren't running if the previous one succeeds (eg `&&`), if the first test command fails and the second one passed wont this mean (for CI) that the tests have passed?

Good point

**Comment by ebebbington** [comment]
> > On the commit `[issue-#175-support-deno-doc] test all of the things` (` 3436713`), because the multiple commands to run the tests aren't running if the previous one succeeds (eg `&&`), if the first test command fails and the second one passed wont this mean (for CI) that the tests have passed?
> 
> Good point

Could chain them together?
```
run one command \
  && run the other \
  && everything checks out
```
Or whatever way looks better, im sure theres a dozen ways to do this

File: src/compilers/template_engine.ts [hunk 1]
```diff
@@ -2,12 +2,32 @@ const decoder = new TextDecoder();
...
+
+  /**
+   * @description
+   *     Construct an object of this class.
+   *
+   * @param number code
+   *     The HTTP response code associated with this exception.
+   * @param string message
+   *     (optional) The exception message.
+   */
```
Comment (ebebbington) [inline src/compilers/template_engine.ts:23]: Little picky.. but doesn't seem to be a `code` param passed in?

Comment (crookse) [reply src/compilers/template_engine.ts:23]: LOL copy paste.

Comment (crookse) [reply src/compilers/template_engine.ts]: fixed

**Pull request merged**